pkg/cli/admin/release/git.go

```diff
@@ -143,20 +148,17 @@ func gitOutputToError(err error, out string) error {
 	return fmt.Errorf(out)
 }
 
-func mergeLogForRepo(g *git, repo string, from, to string) ([]MergeCommit, error) {
+var (
+	rePR     = regexp.MustCompile(`^Merge pull request #(\d+) from`)
+	reBug    = regexp.MustCompile(`^\s*[B|b]ug(?:s)?\s([\d\s,]+)(?:-|:)\s*`)
```

Comment:
will `([\d\s,]+)` match if the number is missing and you put just a `,` or second whitespace?

I guess that's fine if these regexes are not to enforce the schema but only to parse as much as they can from what was already committed

Reply:
I don't like that we are striping leading whitespace, especially since you trimmed the string before you used this regexp.  Can we kill the first `\s*` ?

Reply:
reBug    = regexp.MustCompile(`^\s*[B|b]ug(?:s)?`\s`([\d\s,]+)(?:-|:)\s*`)
  Does this mean that `Bug:123456` wouldn't work? Since there is no space?

Reply:
Can relax and tighten, yes.

Reply:
Bug:153245 hasn't worked no.  Because we only allow dash and colon after the bug number

Reply:
Do you want to allow ` Bug 123:` at all (leading whitespace)?  That's what the `\s*` grants

Reply:
The bot allows leading whitespace (is what I mean)

Reply:
`^[B|b]ugs?\s([\d\s,]+)[:-]\s*`   seems correct and more readable to me.  (no leaving whitespace since that gets cleaned https://github.com/openshift/oc/pull/105/files#diff-17e2ea2595b28107c29a952fdfe5d9cdR210 no?

Reply:
I agree it's more readable, but does the bot block that?  The issue here is the bot being a bit promiscuous and just letting any old PR merge, proper English or no.